kotlin/samples/tictactoe/android/src/main/java/com/squareup/sample/mainactivity/MainActivity.kt

```diff
@@ -43,7 +41,7 @@ class MainActivity : AppCompatActivity() {
 
     workflowRunner = setContentWorkflow(viewRegistry, component.mainWorkflow, savedInstanceState)
         .apply {
-          loggingSub = renderings.subscribe { Timber.d("rendering: %s", it) }
+          renderings.observe(this@MainActivity, Observer { Timber.d("rendering: %s", it) })
```

Comment:
Nit: I think timber has a flag you can check to see if debugging is on, you could gate this observe on that.

Reply:
Noice.